include/deal.II/lac/vector.h

```diff
@@ -152,26 +145,22 @@ class Vector : public Subscriptor
    *
    * We would like to make this constructor explicit, but standard containers
    * insist on using it implicitly.
+   *
+   * @dealiiOperationIsMultithreaded
    */
   Vector(const Vector<Number> &v);
 
   /**
    * Move constructor. Creates a new vector by stealing the internal data of
    * the vector @p v.
    */
-  Vector(Vector<Number> &&v) noexcept;
+  Vector(Vector<Number> &&v) noexcept = default;
 
   /**
    * Copy constructor taking a vector of another data type. This will fail if
    * there is no conversion path from @p OtherNumber to @p Number. Note that
    * you may lose accuracy when copying to a vector with data elements with
    * less accuracy.
-   *
-   * Older versions of gcc did not honor the @p explicit keyword on template
-   * constructors. In such cases, it is easy to accidentally write code that
-   * can be very inefficient, since the compiler starts performing hidden
-   * conversions. To avoid this, this function is disabled if we have detected
-   * a broken compiler during configuration.
```

Comment:
nice catch too!

Reply:
I removed the check itself in #4734 but forgot about the code. Oops.